SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The indicated versions below are currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We acknowledge that every line of code that we write may potentially contain security issues.
+We are committed to address issues responsibly and provide patches as quickly as possible.
+Please contact us via [E-Mail (contact@civicactions.com)](mailto:contact@civicactions.com?subject=Your%20Project:%20Security:=Security%20description:) immediately after discovery of the vulnerability. Thanks!
```

Comment:
Unless this is supposed to be a template, the `mailto:` link should be specific (and `&body=` set correctly, e.g.
`mailto:security@civicactions.com?subject=[GIT%20SECURITY]%20CivicActions%20Guidebook&body=Security%20description`

The `[GIT SECURITY]` prefix is proposed to help with managing incoming emails

Reply:
Thank you @openprivacy. Updated

Reply:
@dmundra - I didn't see the change to the `mailto:` URI which should be:

`[E-Mail (security@civicactions.com)](mailto:security@civicactions.com?subject=[GIT%20SECURITY]%20CivicActions%20Guidebook&body=Security%20description:)`

though the `[GIT SECURITY]` prefix is an optional proposed convention

Reply:
@openprivacy here is the change https://github.com/CivicActions/guidebook/pull/1545/commits/d2e0ae7dece1bf19cabc539fca8c80cba5c114f3, does that look good?